CHANGELOG.md

```diff
@@ -1,3 +1,151 @@
+# 108.0.0
+
+This major release includes several API additions to BottomSheet, Dialogs, and NavigationDrawer, along with a bug fix to Snackbar and improved Buttons documentation.
+
```

Comment:
What was deleted that caused this to be a major version release?

Reply:
`init` was deprecated on MDCSemanticColorScheme. I thought this was a bit borderline, so I'm happy to go down to a minor version if you feel that's appropriate.

Reply:
Ah yeah deprecations are minor releases; deletions are major.

Reply:
Sounds good! Will update.